scripts/Load_collection/load_collection.yml

```diff
@@ -0,0 +1,64 @@
+    type: text
+    example: 'scripts/Load_collection/input/wkt_polygon_test.txt'
+  epsg:
+    label: study area EPSG in
+    description: Numerical value referring to the EPSG code (European Petroleum Survey Group) associated with the spatial reference system that will be used as a reference for the study area. This numerical value specifies the projection and geodetic datum used to define the coordinates and spatial representation of the data in the study area. For further information on coordinate systems and EPSG codes, you can access the official database on the EPSG website at https://epsg.org/home.html. The website provides documentation, resources, and tools for searching and understanding the EPSG codes used in various geospatial contexts
+    type: int
+    example: 3395
+  resolution:
+    label: resolution
+    description: Numerical value referring to the spatial resolution (pixel size) in square meters to create a grid of the study area. This value indicates the level of detail or granularity at which the study area will be analyzed. Each pixel will be considered as a study site for occupancy analyses. A value of 100 represents an hectare and a value of 1000 represents a square kilometer.  
+    type: int
+    example: 1000
+  time_period:
+    label: time interval
+    description: String refering to the periods or intervals used for analysis. It specifies the length and units of time that will be considered for analyzing the data. For example, time intervals like 10Y (10 years), 5Y (5 years), 4M (4 months) can be used to define the time granularity for conducting the analysis. If no interval (time_interval) is specified, '1Y' is used by default
+    type: text
+    example: 'P10Y'
+  time_start:
+    label: Start of time interval
+    description: Refering to the starting date of the time interval. If this value is null or not provided, the minimum date from the column_date will be automatically taken as the starting date for the analysis. This argument is ignored if column_date=NULL
+    type: text
+    example: 'NA'
+  time_end:
+    label: End of time interval
+    description: "Refering to the ending date of the time interval. If this value is null or not provided, the minimum date from the column_date will be automatically taken as the starting date for the analysis. This argument is ignored if column_date=NULL."
+    type: text
+    example: 'NA'
+
+outputs:
+  area_stack:
+    label: area_stack
+    description: area_stack
+    type: image/tiff;application=geotiff
+  dir_stack:
+    label: stack path out
+    description: Path to the output directory where the raster results of the metric analysis are saved
```

Comment:
Can this output be an array of rasters instead?
That way we could show them in the viewer.